backend/de.metas.util/src/main/java/de/metas/util/time/SystemTime.java

```diff
@@ -1,59 +1,26 @@
 package de.metas.util.time;
 
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
 import java.sql.Timestamp;
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
-import java.time.temporal.ChronoUnit;
-import java.util.Calendar;
 import java.util.Date;
 import java.util.GregorianCalendar;
 
-import javax.annotation.Nullable;
-
-/*
- * #%L
- * de.metas.util
- * %%
- * Copyright (C) 2015 metas GmbH
- * %%
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as
- * published by the Free Software Foundation, either version 2 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public
- * License along with this program. If not, see
- * <http://www.gnu.org/licenses/gpl-2.0.html>.
- * #L%
- */
-
-import lombok.NonNull;
-import lombok.experimental.UtilityClass;
-
 /**
- * Code taken from the book "Test Driven", Chapter 7 ("Test-driving the
- * unpredictable") by Lasse Koskela.
- *
- * @author ts
+ * Please consider {@link de.metas.common.util.time.SystemTime}.
```

Comment:
```suggestion
 * @deprecated please consider {@link de.metas.common.util.time.SystemTime}.
```

Reply:
yup... and additionally i will also replace the current usages. Just to not introduce another 600 warnings.

Reply:
update: i've removed that class